app/views/components/common/Quantity.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { StyleSheet, Text } from "react-native";
+
+import colors from "../../styles/colors";
+
+export enum QUANTITY_UNITS {
+  KG = "kg",
+  GM = "gm",
+  ML = "ml",
+  L = "l",
+}
```

Comment:
Should this be added into constants/enums?

Reply:
Yes, will move it over

Reply:
Updated